models/spring-ai-vertex-ai-gemini/src/main/java/org/springframework/ai/vertexai/gemini/VertexAiGeminiChatOptions.java

```diff
@@ -269,6 +274,14 @@ public void setGoogleSearchRetrieval(boolean googleSearchRetrieval) {
 		this.googleSearchRetrieval = googleSearchRetrieval;
 	}
 
+	public List<SafetySetting> getSafetySettings() {
+		return safetySettings;
+	}
+
+	public void setSafetySettings(List<SafetySetting> safetySettings) {
```

Comment:
Please add non-null assertion check.

Reply:
@tzolov Other properties have their assertions on the Builder, not the Options itself (see `mimeType` and `functions`). Would you prefer to favor consistency in this case or set it on the Options class?

Reply:
I added the check in both places. Let me know if you'd like to favor consistency and I'll remove it from the class setter.